server/filestore.go

```diff
@@ -4080,16 +4083,16 @@ func (fs *fileStore) loadLast(subj string, sm *StoreMsg) (lsm *StoreMsg, err err
 
 // LoadLastMsg will return the last message we have that matches a given subject.
 // The subject can be a wildcard.
-func (fs *fileStore) LoadLastMsg(subject string, sm *StoreMsg) (*StoreMsg, error) {
+func (fs *fileStore) LoadLastMsg(subject string, smv *StoreMsg) (sm *StoreMsg, err error) {
```

Comment:
But changing `sm` to `smv` in function parameters, we are not using `smv` at all, so that seem wrong.

Reply:
Good catch, will fix.